.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+on:
+  push:
+
+jobs:
+  jvm:
+    strategy:
+      fail-fast: true
+      matrix:
+        java: [11, 16]
```

Comment:
Just java 17 perhaps? 16 wasn't a LTS release

Reply:
Yeah, I tried Java 17 but [AdoptOpenJDK](https://adoptopenjdk.net/) doesn't support it as yet. We could try and use another Java provider?

Reply:
I managed to get `openjdk` working with Java 17.